tests/test_main.py

```diff
@@ -1144,36 +1144,9 @@ class SubUser(User):
     assert not hasattr(t.user, 'sins')
 
 
-@pytest.mark.skip(reason='not implemented')
-@pytest.mark.parametrize('comv_value', [True, False])
-def test_copy_on_model_validation_warning(comv_value):
```

Comment:
This replaced by:

- (test_revalidate_instances_never)[https://github.com/pydantic/pydantic/blob/ca8069811a30c271c9ee0b36d455c9c0adb458a0/tests/test_main.py#L1070]
- (test_revalidate_instances_sub_instances)[https://github.com/pydantic/pydantic/blob/ca8069811a30c271c9ee0b36d455c9c0adb458a0/tests/test_main.py#L1095]
- (test_revalidate_instances_always)[https://github.com/pydantic/pydantic/blob/ca8069811a30c271c9ee0b36d455c9c0adb458a0/tests/test_main.py#L1121]